metadata/site_import/services.xml

```diff
@@ -30,6 +30,11 @@
         <user-id></user-id>
         <password/>
     </service-credential>   
+    <service-credential service-credential-id="AdyenOriginKeys">
+        <url>https://checkout-test.adyen.com/v40/originKeys</url>
```

Comment:
in order to not create a new service-credential for every api call, can't we modify the service factory to append these URLs?
on https://github.com/Adyen/adyen-salesforce-commerce-cloud/blob/36ac2b7cf76586d0dbcda0ae38e59408249c0a14/cartridges/int_adyen/cartridge/scripts/util/AdyenHelper.ds#L41
to do svc.setUrl